engines/sci/engine/file.cpp

```diff
@@ -126,15 +242,7 @@ reg_t file_open(EngineState *s, const Common::String &filename, int mode, bool u
 		return SIGNAL_REG;
 	}
 
-	// Find a free file handle
-	uint handle = 1; // Ignore _fileHandles[0]
-	while ((handle < s->_fileHandles.size()) && s->_fileHandles[handle].isOpen())
-		handle++;
-
-	if (handle == s->_fileHandles.size()) {
-		// Hit size limit => Allocate more space
-		s->_fileHandles.resize(s->_fileHandles.size() + 1);
-	}
+	uint handle = findFreeFileHandle(s);
```

Comment:
There is some `VIRTUALFILE_HANDLE_START` and `VIRTUALFILE_HANDLE_END` a bit below here at `getFileFromHandle` (line 255) but I don’t actually see anything that would give a handle in that range. Is this a game script thing, or just dead code?